client/src/components/AppointmentTable.js

```diff
@@ -21,13 +26,13 @@ const AppointmentTable = () => {
       );
 
     return data.appointments.map(({ id, startTime, endTime, location }) => (
-      <tr key={id}>
+      <tr key={id} onClick={() => goToEditAppointmentPage(id)}>
         <td>
           {moment.unix(startTime).format("LLL")} -{" "}
           {moment.unix(endTime).format("LLL")}
         </td>
         <td>{location.address}</td>
-        <td></td>
+        <td>{id}</td>
```

Comment:
Why do we have to show IDs in a column?

Reply:
I just deleted it. It was just a way handy for me to check if I passed the correct {id} to the relating url.